rpa-builder/6.6/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc

```diff
@@ -8,7 +8,7 @@ to decide which activity should be executed next.
 
 image:toolbox-activity-parameters.png[The Activity Parameters Action Step, 40%, 40%]
 
-Only one _Activity Parameters_ Action Step can exist in a process and it is automatically placed in the _Workflow Initialization_ section when creating a new Workflow.
+Only one _Activity Parameters_ Action Step can exist in a process and it's automatically placed in the _Workflow Initialization_ section when creating a new Workflow.

```

Comment:
```suggestion
A process can have one _Activity Parameters_ Action Step and it's automatically placed in the _Workflow Initialization_ section when creating a new Workflow.
```

Reply:
Replaced 'process' with 'workflow', because a process can contain seberal workflows and each of them can contain one AP AS